### DISCUSSION TITLE
Welcome Peloton Riders

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#4143] [Peloton]
# 🚧 The Peloton is improving. Watch this space 🚧

# Peloton

<p align="center">
<a href="https://github.com/SciTools/iris/issues?q=is%3Aissue+is%3Aopen+label%3A%22Peloton+%F0%9F%9A%B4%E2%80%8D%E2%99%82%EF%B8%8F%22">
    <img src="https://img.shields.io/github/issues/Scitools/iris/Peloton%20%F0%9F%9A%B4%E2%80%8D%E2%99%82%EF%B8%8F?color=yellow&label=Peloton%20%F0%9F%9A%B4%20%7C%20Issues&style=plastic" alt="GitHub peloton issues"></a>
<a href="https://github.com/SciTools/iris/pulls?q=is%3Apr+is%3Aopen+label%3A%22Peloton+%F0%9F%9A%B4%E2%80%8D%E2%99%82%EF%B8%8F%22">
    <img src="https://img.shields.io/github/issues-pr/SciTools/iris/Peloton%20%F0%9F%9A%B4%E2%80%8D%E2%99%82%EF%B8%8F?color=yellow&label=Peloton%20%F0%9F%9A%B4%20%7C%20Pull-Requests" alt="GitHub peloton prs"></a>
</p>

Have some free time on your hands? Well you're more than welcome to ride with the `iris` Peloton. The more the merrier!

So don your best Lycra and crash helmet, as it's time to catch some breakaway `iris` issues and pull-requests :biking_man::biking_woman: 

Once you've eased yourself into the saddle, checkout our [Peloton Kanban Board](https://github.com/SciTools/iris/projects/29) for open issues and pull-requests
- **Own It** - simply **assign** yourself to an open Peloton ticket from the **Backlog** column
- **Do It** - drag the ticket into the **In Progress** column, hunker down and get peddling
- **Close It** - see the issue or pull-request through to the finishing line by closing or merging

The Peloton is stronger than its individual riders. So if you need help, simply ask by using @SciTools/peloton in a comment to grab the attention of other riders.

Thank you! See you on the road! :heart: 

![Tour-de-France-dans-le-peloton-un-grand-ouf-de-soulagement](https://user-images.githubusercontent.com/2051656/118801535-08ad1500-b899-11eb-9986-8b841e51eca9.jpg)

**Comment by rcomer** [comment]
What defines a "breakaway" issue or PR?

**Reply by trexfeathers** [reply]
I suppose it's any issue/PR that we think has been open for too long.

It's a bit of a stretched metaphor (which I still like). In cycling races the main Peloton can only catch a breakaway group by working together (due to aerodynamics), but they don't always choose to try (multi-day tactics). So I understand @bjlittle's metaphor to mean we (the Peloton) have chosen to work together to overcome 1 or more issues/PR's (breakaway groups) by working together.

**Reply by pp-mo** [reply]
🚴 👈  Stop him, he's getting away !

**Reply by bjlittle** [reply]
@rcomer Do you fancy coming along to a peloton? We hangout for no more than 30 minutes.

You might enjoy it :smile:

**Reply by rcomer** [reply]
@bjlittle oh go on then!

**Comment by rcomer** [comment]
This discussion is currently pinned, but doesn't reflect how the Peloton works any more.  Should we

a) unpin it
b) modify the OP
c) archive it
d) two or more of the above

?
